lib/services/AutoRenewService.ts

```diff
@@ -67,6 +67,7 @@ export class AutoRenewService implements ServiceInterface {
     if (this.canStart()) {
       await this.stop();
       this.tokenManager.on(EVENT_EXPIRED, this.onTokenExpiredHandler);
+      this.tokenManager.resetExpireEventTimeoutAll();
```

Comment:
I was hoping having `serviceManager.start` await on the leadership election would eliminate the need for this call, is that not the case?